### ISSUE TITLE
Expected version 128, instead got 38

### ISSUE THREAD

**Issue by osirisr** [steemit/steem-js#348]
Simply trying to make a post, and get the error code `Expected version 128, instead got 38`

Does anyone know what this means? All parameters including WIF are correct, and it was working before using the browser method. Currently using the node.js method (running a node.js server).

**Comment by ohrak22gmail** [comment]
i have same problem

**Comment by ohrak22gmail** [comment]
I solved this problem.
using private posting key.

**Comment by yashwanth2804** [comment]
yes , in my case i given entire permalink http instead of post title

**Comment by nickberns** [comment]
Encountered this too. Needs to be documented. Solution is to use private posting key.

**Comment by ety001** [comment]
This repository has been superseded by a complete 2025 rewrite. The new trunk (`master`, starting at v1.1.0) is a ground-up rewrite of `@steemit/steem-js` and the legacy 0.7.x / 0.8.x code line is archived.

As part of that transition, this issue is being closed because:
- It was filed against the legacy codebase, and the code it references no longer exists in the rewritten trunk.
- The new codebase reimplements the affected areas (RPC routing, transaction serialization, auth/signing, crypto, browser builds) from scratch, so this issue cannot be carried over as-is.

If the problem or request still applies to the rewritten codebase (v1.1.0+ on `master`), please open a new issue against the current trunk with details that reproduce on the new version. Thank you for taking the time to report this originally.
